File: javascript/p002.js
function main() {
	let total = 0;
	let a = 1;
	let b = 2;
	while (a <= 4000000) {
		if (a % 2 === 0) {
			total += a;
		};
		let c = a + b;
		a = b;
		b = c;
	};
	return total;
};

console.log(main());

File: python/p002.py
from funcs import isPrime

def main():
	# previous fibonacci number, current fibonacci number
	x, y = 1, 2
	total = 0
	while x <= 4e6:
		# checking if x is even
		if x % 2 == 0:
			total += x
		z = x + y
		x = y
		y = z
	return total
print(main())